client/consensus/grandpa/auxSchema.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
+import (
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"golang.org/x/exp/constraints"
+
+	finalityGrandpa "github.com/ChainSafe/gossamer/pkg/finality-grandpa"
+)
+
+var (
+	SET_STATE_KEY      = []byte("grandpa_completed_round")
+	CONCLUDED_ROUNDS   = []byte("grandpa_concluded_rounds")
+	AUTHORITY_SET_KEY  = []byte("grandpa_voters")
+	BEST_JUSTIFICATION = []byte("grandpa_best_justification")
+)
+
+type databaseKeyValue [2][]byte
+
+// SharedVoterSetState .
+type SharedVoterSetState[H comparable, N constraints.Unsigned] struct {
+	// TODO move and implement
+	inner VoterSetState[H, N]
+}
+
+type PersistentData[H comparable, N constraints.Unsigned] struct {
+	authoritySet SharedAuthoritySet[H, N] // TODO this needs to be shared auth set
+	setState     SharedVoterSetState[H, N]
+}
+
+func loadPersistent[H comparable, N constraints.Unsigned](store AuxStore, genesisHash H, genesisNumber N, genesisAuthorities AuthorityList) (*PersistentData[H, N], error) {
+	genesis := finalityGrandpa.HashNumber[H, N]{Hash: genesisHash, Number: genesisNumber}
+	makeGenesisRound := finalityGrandpa.NewRoundState[H, N]
+
+	// No longer use version to tell if data is persisted, just use actual data
```

Comment:
revise this comment.  We never ported over different versions in the first place.